src/VisionModel/googlenet.jl

```diff
@@ -0,0 +1,59 @@
+using Flux
+
+export googlenet
+
+function conv_block(kernelsize::Tuple{Int64,Int64}, inplanes::Int64, outplanes::Int64; stride::Int64=1, pad::Int64=0)
```

Comment:
This function definition always calls with ``stride=1``. I think changing this would help. But ``stride=0`` shows ``Integer division error``.

Reply:
The stride can never be 0.

Reply:
yes, but I am getting a dimension mismatch error as above